sourced/ml/engine.py

```diff
@@ -30,3 +31,15 @@ def create_engine(session_name, repositories, kwargs):
     log.info("Initializing on %s", repositories)
     engine = Engine(session, repositories)
     return engine
+
+
+def get_tokens(uasts):
```

Comment:
No, this should go here. This file contains only the functions to launch the engine and its dependencies. Tokens extraction is a super narrow problem.

Besides, this duplicates with `uast_ids_to_bag.py` and not needed.

Reply:
I will delete it, but `uast_ids_to_bag.py` is outdated. Will reimplement it using new engine later.

Reply:
It is not outdated. It contains the single payload class `UastIds2Bag` and we use it in many places. Please do not touch it.